puts/vendor/configs/openssl.toml

```diff
@@ -0,0 +1,52 @@
+[preset]
```

Comment:
This is really cool :)

Reply:
Glad you like it 😄. Hopefully in the future adding support for new configurations will be as easy as adding a few lines here 🥲